analytics-recordings.php

```diff
@@ -103,21 +103,25 @@
     $wheresql .= " AND bigbluebuttonbnid = :bigbluebuttonbnid";
     $params['bigbluebuttonbnid'] = $bigbluebuttonbn->id;
 }
-$table->set_sql('id, meta, userid, timecreated', "{bigbluebuttonbn_logs}", $wheresql, $params);
 $table->define_baseurl($pageurl);
 
 // Add cloned table modifying the query for Past Month instead.
 $pastmonthtable = clone $table;
 $wheresqlwithmonthconstraint = $wheresql;
-$wheresql .= " AND timecreated > :timecreated";
+$wheresqlwithmonthconstraint .= " AND timecreated > :timecreated";
 $params['timecreated'] = time() - (30 * DAYSECS); // Currently set to 1 calendar month.
-$pastmonthtable->set_sql('id, meta, userid, timecreated', "{bigbluebuttonbn_logs}", $wheresql, $params);
+$pastmonthtable->set_sql('id, meta, userid, timecreated', "{bigbluebuttonbn_logs}", $wheresqlwithmonthconstraint, $params);
 $pastmonthtable->setup();
 $pastmonthtable->query_db(0); // No limit, fetch all rows.
 $pastmonthtable->pageable(false);
 $pastmonthtable->close_recordset();
 $pastmonthmeetingcreateevents = $pastmonthtable->rawdata;
 
+// Update main table query with time filter
+$wheresql .= " AND timecreated > :timecreated";
```

Comment:
It seems we can just do this at L111 and remove $wheresqlwithmonthconstraint entirely, as its only the parameter passed in that needs to change

Reply:
Did you one better, please see changes.